src/main/java/beam/matsim/CustomPlansDumpingImpl.java

```diff
@@ -0,0 +1,76 @@
+package beam.matsim;
```

Comment:
Same - I don't see this used, but is probably part of matsim injection?

Reply:
correct, it is used [here](https://github.com/LBNL-UCB-STI/beam/pull/2151/files?file-filters%5B%5D=.conf&file-filters%5B%5D=.scala#diff-d4f57c2659d44575c5f00b19c7e0449cR159)